### PR TITLE
Implemented clean_twisted_observers method

### PR DESCRIPTION
**Pull request by devos50**

### CODE REVIEW

File: twisted/plugins/tracker_plugin.py [hunk 1]
```diff
@@ -50,6 +52,10 @@
 else:
     SOCKET_BLOCK_ERRORCODE = errno.EWOULDBLOCK
 
+# Remove the limited history observer log
```
Comment (whirm) [inline twisted/plugins/tracker_plugin.py]: maybe move this to the other module and add a link to the issue so we can track its evolution?

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
@whirm tests succeeded but build is unstable

**Comment by whirm** [comment]
https://jenkins.tribler.org/job/GH_Dispersy_pull-request-tester_devel/911/violations/

**Comment by devos50** [comment]
@whirm want me to fix them?

**Comment by whirm** [comment]
at least the logging stuff

**Comment by whirm** [comment]
```
20              logger.debug("Removing observer", o
21  
22          else:
23              logger.debug("Leaving alone observer", o)
```

**Comment by whirm** [comment]
have you checked that the twisted plugin you modified still runs?

**Comment by devos50** [comment]
@whirm this is covered in a test: https://github.com/Tribler/dispersy/blob/devel/tests/test_bootstrap.py#L32

**Comment by devos50** [comment]
Also, I've fixed the logging (didn't see the wrong formatting, thanks!)

**Pull request merged**